management_os/programs/fs.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"errors"
+	"os"
+)
+
+// Size with which to copy
+const blocksize int64 = 512
+
+// CopyFile is a function which copies a file, it is similar to dd in usage
+func CopyFile(from, to string) error {
+
+	src, err := os.OpenFile(from, os.O_RDONLY, os.ModePerm)
```

Comment:
If this ends up being to slow we could use [mmap](https://pkg.go.dev/golang.org/x/exp/mmap) for a potential speedup

Reply:
It only needs to be faster than the networking right? Or do you intend to use it for other purposes? If so we'll just have to wait on how fast we can get the networking to be.